.github/workflows/ci.yml

```
@@ -198,7 +198,7 @@ jobs:
           pipenv install --dev --ignore-pipfile --python ${{ matrix.python-version }}
 
       - name: Check migrations
-        if: needs.changed-files.outputs.run_tests == 'true'
+        if: ${{ needs.changed-files.outputs.run_tests == 'true' && !(contains(github.event.pull_request.labels.*.name, 'skip-migration-check')) }}
```

Using the [literal block scalar](https://yaml.org/spec/1.2.2/#812-literal-style) makes this more readable.
```suggestion
        if: |
          needs.changed-files.outputs.run_tests == 'true'
          && !(contains(github.event.pull_request.labels.*.name, 'skip-migration-check'))
```